cranelift/filetests/filetests/runtests/atomic-load-store.clif

```diff
@@ -0,0 +1,105 @@
+test interpret
+test run
+set enable_llvm_abi_extensions=true
+target x86_64
+target aarch64
+target riscv64
+target s390x
+
+function %i64_atomic_store_load(i64) -> i8 {
+    ss0 = explicit_slot 8
+
+block0(v0: i64):
+    v1 = stack_addr.i64 ss0
+    atomic_store.i64 v0, v1
+    v2 = atomic_load.i64 v1
+
+    v3 = icmp.i64 eq v0, v2
+    return v3
```

Comment:
What would you think of returning the result of `atomic_load` (`v2`) directly and letting the test harness check equality, instead of relying on `icmp` to generate correct code?